src/region/RegionCreator.tsx

```diff
@@ -255,7 +262,7 @@ export default function RegionCreator({ className, onAbort, onStart, onStop }: P
             {drawingStatus === DrawingStatus.drawing && (
                 <RegionRect ref={regionRectRef} className="ba-RegionCreator-rect" isActive />
             )}
-            {drawingStatus === DrawingStatus.init && isHovering && <PopupCursor />}
+            {drawingStatus === DrawingStatus.init && isHovering && !isFtuxCursorDisabled && <PopupCursor />}
```

Comment:
This seems like a lot of effort to through to check a single condition. Can we hide the cursor with CSS somehow from the Preview SDK? Or from the individual annotators?

Reply:
Yes I think that approach would be a lot simpler, I can go ahead and close this PR as it's no longer needed.